[opdocs] Kommander: Split clusters into multiple pages (#3400)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/clusters/edit-cluster/index.md

---
layout: layout.pug
beta: true
navigationTitle: Editing Clusters
title: Editing Clusters
menuWeight: 7
excerpt: Edit a cluster
---

## Edit a cluster

![Edit a Cluster Action](/dkp/kommander/1.3/img/edit-cluster-action.png)

### Edit an attached cluster

> For an attached cluster, you can only edit labels that are assigned to it.

Reviewer: ```suggestion
For an attached cluster, you can only edit labels assigned to that cluster.
```

![Edit an Attached Cluster](/dkp/kommander/1.3/img/edit-cluster-attached-1-1-0.png)

### Edit a managed cluster

> For a managed cluster, you can edit its name, Kubernetes version, labels, cloud provider tags, and its node pools.

Reviewer: ```suggestion
For a managed cluster, you can edit the following:
- Name
- Kubernetes version
- Labels
- Cloud provider tags
- Node pools.
```

![Edit a Cluster Form](/dkp/kommander/1.3/img/edit-cluster-form-name-1-1-0.png)

#### Edit a node pool

> When editing a node pool, you can only increase the number of nodes in the pool. This is to prevent losing any workloads that are currently running on the cluster.

Reviewer: ```suggestion
When editing a node pool, you can only increase the number of nodes in the pool. This prevents losing any workloads currently running on the cluster.
```
